sign_language_segmentation/bin.py

```diff
@@ -59,15 +60,36 @@ def predict(model, pose: Pose):
 
 def get_args():
     parser = argparse.ArgumentParser()
-    parser.add_argument('--pose', required=True, type=str, help='path to input pose file')
+    parser.add_argument('--pose', required=True, type=Path, help='path to input pose file')
     parser.add_argument('--elan', required=True, type=str, help='path to output elan file')
+    parser.add_argument('--save_crops', 
```

Comment:
for options, please use kebab-case (`--save-crops`) for for this one i recommend the naming `--save-segments`

Reply:
That does seem like a better name. Fixing.